ethereum/gateway-eth-ts/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "precompile": "rm -rf dist",
     "compile": "tsc -p tsconfig.json",
-    "lint": "eslint lib/ src/ test/ --ext .ts",
```

Comment:
Do we not want to lint those folders or does this come in a follow up PR?

Reply:
Having these folders in the command makes it fail because there are no files in test/ and there is no lib/

Reply:
oh that's fine then. Confused there were not tests then :)